sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeHealthcareEntitiesTests.cs

```diff
@@ -496,7 +498,7 @@ private void ValidateBatchDocumentsResult(AnalyzeHealthcareEntitiesResultCollect
         private void ValidateOperationProperties(AnalyzeHealthcareEntitiesOperation operation)
         {
             Assert.AreNotEqual(new DateTimeOffset(), operation.CreatedOn);
-            Assert.AreNotEqual(new DateTimeOffset(), operation.LastModified);
+            // Assert.AreNotEqual(new DateTimeOffset(), operation.LastModified);
```

Comment:
I will create an issue to keep track of this one. For now, the new 2022-10-01-preview version has a regression with this field that the service team is taking care of.